doc/newsfragments/2117_changed.interactive_uid.rst

```diff
@@ -0,0 +1 @@
+Reset uid when saving report in interactive mode.
```

Comment:
This is a user facing releasenote, this does not tell to much to the user. Can it be like: "Fixed an issue where report saved from interactive mode having problem with testcases having the same name" or similar, that describe what the user will see?